addon/templates/components/validated-input.hbs

```diff
@@ -38,18 +38,19 @@
   {{#if (eq type "select")}}
 
     {{one-way-select
-      value            = (or value (get model name))
-      options          = options
-      optionLabelPath  = optionLabelPath
-      optionValuePath  = optionValuePath
-      optionTargetPath = optionTargetPath
-      name             = name
-      class            = config.css.control
-      update           = (action "update")
-      focusOut         = (action "setDirty")
-      includeBlank     = includeBlank
-      disabled         = disabled
-      multiple         = multiple
+    value               = (or value (get model name))
+    options             = options
+    optionLabelPath     = optionLabelPath
+    optionValuePath     = optionValuePath
+    optionTargetPath    = optionTargetPath
+    name                = name
+    class               = "form-control"
+    update              = (action "update")
+    focusOut            = (action "setDirty")
+    prompt              = (if includeBlank (or includeBlank null))
+    disabled            = disabled
+    multiple            = multiple
+    promptIsSelectable  = (if promptIsSelectable (or promptIsSelectable false))
```

Comment:
The reason I added promptIsSelectable in this way was to maintain backwards compatibility with any existing implementations, but I'm happy to change it.